crates/edr_eth/src/transaction.rs

```diff
@@ -19,6 +19,7 @@ pub use self::{detailed::DetailedTransaction, kind::TransactionKind, request::*,
 #[cfg_attr(feature = "serde", serde(deny_unknown_fields))]
 #[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
 pub struct EthTransactionRequest {
+    // TODO review question: should this be non-optional? In the `eth_sendTransaction` docs and in the Hardhat `rpcTransactionRequest` type, `from` is marked as compulsory.
```

Comment:
Review question

Reply:
I think you are right. Feel free to remove the option

Reply:
Done in [Make from in EthTransactionRequest mandatory](https://github.com/NomicFoundation/hardhat/pull/4548/commits/818542e5a42893c2f6235218cc58de79897246e1)